src/databricks/labs/ucx/workspace_access/listing.py

```diff
@@ -60,6 +72,9 @@ def _list_and_analyze(self, obj: ObjectInfo) -> (list[ObjectInfo], list[ObjectIn
         except NotFound:
             # See https://github.com/databrickslabs/ucx/issues/230
             logger.warning(f"{obj.path} is not listable. Ignoring")
+        except TimeoutError as e:
+            logger.warning(f"Could not list {obj.path} due to backend error. Ignoring")
+            logger.warning(e)
```

Comment:
```suggestion
            logger.warning(f"Could not list {obj.path} due to backend error: {e}")
```
don't do two log lines for one event.